applications/StructuralMechanicsApplication/custom_response_functions/adjoint_processes/replace_elements_and_conditions_for_adjoint_problem_process.cpp

```diff
@@ -13,6 +13,7 @@
 // Project includes
 #include "utilities/compare_elements_and_conditions_utility.h"
 #include "replace_elements_and_conditions_for_adjoint_problem_process.h"
+#include "structural_mechanics_application_variables.h"
```

Comment:
intentional?

Reply:
Yes, i guess i removed an include in one of the here included files, so now it was missing...

Reply:
it was because the "shell_thin_element_3D3N.h" is not included anymore in the adjoint shell element.